packages/back-end/src/routers/event-webhooks/event-webhooks.controller.ts

```diff
@@ -56,12 +55,14 @@ export const getEventWebHook = async (
   req: GetEventWebHookByIdRequest,
   res: Response<GetEventWebHookByIdResponse | PrivateApiErrorResponse>
 ) => {
-  req.checkPermissions("manageWebhooks");
```

Comment:
Same here - For all other globally-scoped resources, we don't filter them by user read access, so keeping that pattern here.